plugins/cloud-costs/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.3.12
```

Comment:
[opt] [new functionality](https://semver.org/#summary) ideally should be a minor bump